recipes/cylp/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "cylp" %}
+{% set version = "0.92.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/cylp-{{ version }}.tar.gz
+  sha256: 9c4f686964a577558b5c7f637d19657e6364826c22c5cc3fe1d7bebb61961bd6
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - cython
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+  host:
+    - python
+    - setuptools
+    - cython ==3.0.10
```

Comment:
```suggestion
    - cython
```

Is that pin needed?

Reply:
not indicated in PKG-INFO. I don't know why greyskull pinned that.
I think it's not needed.